File: elecciones_app_movil/lib/data/model/candidato/movimiento.dart
class Movimiento {
  Movimiento({
    this.id,
    this.nombre,
    this.siglas,
    this.numero,
    this.fotoArchivo,
    this.fotoNombre,
  });

  int? id;
  String? nombre;
  String? siglas;
  String? numero;
  dynamic fotoArchivo;
  String? fotoNombre;

  Movimiento copyWith({
    int? id,
    String? nombre,
    String? siglas,
    String? numero,
    dynamic fotoArchivo,
    String? fotoNombre,
  }) =>
      Movimiento(
        id: id ?? this.id,
        nombre: nombre ?? this.nombre,
        siglas: siglas ?? this.siglas,
        numero: numero ?? this.numero,
        fotoArchivo: fotoArchivo ?? this.fotoArchivo,
        fotoNombre: fotoNombre ?? this.fotoNombre,
      );

  factory Movimiento.fromMap(Map<String, dynamic> json) => Movimiento(
        id: json["id"],
        nombre: json["nombre"],
        siglas: json["siglas"],
        numero: json["numero"],
        fotoArchivo: json["fotoArchivo"],
        fotoNombre: json["fotoNombre"],
      );

  Map<String, dynamic> toMap() => {
        "id": id,
        "nombre": nombre,
        "siglas": siglas,
        "numero": numero,
        "fotoArchivo": fotoArchivo,
        "fotoNombre": fotoNombre,
      };
}

File: elecciones-api-auth/src/main/java/com/eleccciones/cliente/data/entities/candidatos/Movimiento.java
package com.eleccciones.cliente.data.entities.candidatos;

import com.eleccciones.cliente.data.entities.core.impl.AbstractNamed;
import lombok.Getter;
import lombok.Setter;
import org.hibernate.annotations.Type;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Lob;
import javax.persistence.Table;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "movimiento")
@Getter
@Setter
public class Movimiento extends AbstractNamed {

    @NotNull
    @Column(name = "siglas")
    private String siglas;

    @Column(name = "numero")
    private String numero;

    @Lob
    @Type(type="org.hibernate.type.ImageType")
    @Column(name = "foto_archivo")
    private byte[] fotoArchivo;

    @Column(name = "foto_nombre")
    private String fotoNombre;

}
